src/Form/Form.js

```diff
@@ -207,17 +207,17 @@ class Form {
             }
         }
 
-        // TODO: on mobile, focus could open keyboard before tooltip
-        ['mousedown', 'focus']
-            .forEach((ev) => this.addListener(input, ev, handler, true))
+        const events = ['pointerdown']
+        if (!isMobileApp) events.push('focus')
+        events.forEach((ev) => this.addListener(input, ev, handler, true))
```

Comment:
Had to remove the `focus` listener on mobile because it can cause weird behaviour with the keyboard.